### PR TITLE
Add PR Labeller Workflow

### PR DESCRIPTION
**Pull request by Gauthamastro**

### CODE REVIEW

**Comment by ksr30** [comment]
its failing @Gauthamastro

File: .github/label.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  pull_request:
+    branches: [ develop ]
+
+jobs:
+  triage:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/labeler@main
+        with:
+          repo-token: "${{ secrets.POLKADEX }}"
```
Comment (ksr30) [inline .github/label.yml]: @Gauthamastro insted of POLKADEX use secrets.GITHUB_TOKEN

Comment (ksr30) [reply .github/label.yml]: it's pre-defined

**Review by ksr30 — CHANGES_REQUESTED** [review]

File: .github/label.yml (hunk 1 above)
Comment (Gauthamastro) [reply .github/label.yml]: But GITHUB_TOKEN is not stored in the secrets. Where will it get value then?

Comment (Gauthamastro) [reply .github/label.yml]: It's still failing

**Pull request closed without merging**